rosie/chamber_of_deputies/settings.py

```diff
@@ -0,0 +1,20 @@
+from rosie.chamber_of_deputies.classifiers.election_expenses_classifier import ElectionExpensesClassifier
+from rosie.chamber_of_deputies.classifiers.meal_price_outlier_classifier import MealPriceOutlierClassifier
+from rosie.chamber_of_deputies.classifiers.monthly_subquota_limit_classifier import MonthlySubquotaLimitClassifier
+from rosie.chamber_of_deputies.classifiers.traveled_speeds_classifier import TraveledSpeedsClassifier
+from rosie.chamber_of_deputies.classifiers.irregular_companies_classifier import IrregularCompaniesClassifier
+from rosie.core.classifiers.invalid_cnpj_cpf_classifier import InvalidCnpjCpfClassifier
+
+
+CLASSIFIERS = {
+    'meal_price_outlier': MealPriceOutlierClassifier,
+    'over_monthly_subquota_limit': MonthlySubquotaLimitClassifier,
+    'suspicious_traveled_speed_day': TraveledSpeedsClassifier,
+    'invalid_cnpj_cpf': InvalidCnpjCpfClassifier,
+    'election_expenses': ElectionExpensesClassifier,
+    'irregular_companies_classifier': IrregularCompaniesClassifier
+}
+
+UNIQUE_IDS = ['applicant_id', 'year', 'document_id']
+
+VALUE = 'total_net_value'
```

Comment:
This list of columns will quickly grow large. `total_net_value`, `cnpj_cpf`, `subquota_id` already have different names in Chamber of Deputies and Federal Senate. I'd rather prefer to keep in the `settings.py` a dictionary with dataset_column -> column expected by Rosie, as in:

```python
COLUMNS = {
  'value': 'total_net_value',
  'company_id': 'cnpj_cpf',
}
```

In the case of this example, I suggest to have `rosie.chamber_of_deputies.adapter.Adapter` renaming the dataset columns and the classifiers look for columns named `'value'` and `'company_id'`.